### ISSUE TITLE
AutoComplete control is updated with an outdated value onBlur when filterable true

### ISSUE THREAD

**Issue by HobbsB** [telerik/kendo-angular#1658]
## I'm submitting a...

* Bug report 

## Current behavior

Highlighting a suggestion and then pressing tab (or clicking away) causes the ValueChange event to emit an outdated value (the value the control had when focused) and the control to be updated with it.

## Expected behavior

Textbox value should not change and ValueChange should emit the current value 

## Minimal reproduction of the problem with instructions

https://next.plnkr.co/edit/Im0hI23EgcBr0wn7?preview

Using the AutoComplete component with filterable as "true"

1) Type "Z" in the textbox 
2) click away from the textbox
3) Focus on the textbox
4) delete existing value and type "A"
5) Press arrow key down to highlight the first option
6) Press TAB to blur the field

Expected: Value should remain as "A"
Actual: Value is replaced with the old value "Z"

## Environment

Package versions:

```
@angular-devkit/build-angular                0.6.3   0.6.8   0.6.8  web
@angular/animations                          6.0.2   6.0.7   6.0.7  web
@angular/cli                                 6.0.3   6.0.8   6.0.8  web
@angular/common                              6.0.2   6.0.7   6.0.7  web
@angular/compiler                            6.0.2   6.0.7   6.0.7  web
@angular/compiler-cli                        6.0.2   6.0.7   6.0.7  web
@angular/core                                6.0.2   6.0.7   6.0.7  web
@angular/forms                               6.0.2   6.0.7   6.0.7  web
@angular/http                                6.0.2   6.0.7   6.0.7  web
@angular/language-service                    6.0.2   6.0.7   6.0.7  web
@angular/platform-browser                    6.0.2   6.0.7   6.0.7  web
@angular/platform-browser-dynamic            6.0.2   6.0.7   6.0.7  web
@angular/router                              6.0.2   6.0.7   6.0.7  web
@progress/kendo-angular-buttons              4.1.0   4.1.2   4.1.2  web
@progress/kendo-angular-dateinputs           3.3.0   3.4.2   3.4.2  web
@progress/kendo-angular-dialog               3.6.1   3.6.2   3.6.2  web
@progress/kendo-angular-dropdowns            3.0.0   3.0.2   3.0.2  web
@progress/kendo-angular-excel-export         2.0.0   2.1.1   2.1.1  web
@progress/kendo-angular-grid                 3.3.0   3.5.1   3.5.1  web
@progress/kendo-angular-inputs               3.1.1   3.2.0   3.2.0  web
@progress/kendo-data-query                   1.3.1   1.4.0   1.4.0  web
@progress/kendo-drawing                      1.5.5   1.5.6   1.5.6  web
@progress/kendo-theme-bootstrap             2.13.4  2.13.6  2.13.6  web
```

Browser:
- Chrome (desktop) version XX

**Comment by dimitar-pechev** [comment]
fixed in kendo-angular-dropdowns v3.1.1-dev.201810021437

**Comment by ipeshev** [comment]
Fixed in v3.1.1-dev.201810110855

**Comment by dimitar-pechev** [comment]
Shipped in v3.2.0 of the `@progress/kendo-angular-dropdowns` package.
